Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Agents"
 uuid = "46ada45e-f475-11e8-01d0-f70cc89e6671"
 authors = ["Ali Vahdati", "George Datseris", "Tim DuBois"]
-version = "3.5"
+version = "3.5.1"
```

Comment:
I'm working on #281 at the moment. We may need to bump to 3.6 with that.

Reply:
Alright, then let's first merge #281 and then I'll rebase this one to the master

Reply:
Think it'll be fine with a patch bump with this new implementation.